source/roadmap.html.erb

```diff
@@ -59,49 +59,52 @@ title: Roadmap
             </div>
              <h2 class="govuk-heading-m">Recently released</h2>
               <ul class="govuk-list govuk-list--bullet">
-                <li>3D Secure 2 support for Worldpay and ePDQ</li>
-                <li>View payments Stripe has made into your bank account</li>
-                <li>View and download all transactions from GOV.UK Pay services in one report</li>
-                <li>Add reporting columns to payment links, e.g. cost centre codes</li>
-                <li>CSV download limit increased from 10,000 to 100,000 transactions</li>
-                <li>Making it easier to handle card payments taken by agents over the phone (also known as <a class="govuk-link" href="https://docs.payments.service.gov.uk/moto_payments/#take-a-payment-over-the-phone-moto-payments">MOTO payments</a>)</li>
                 <li>Making GOV.UK Pay more accessible for users</li>
-                <li>Trialling PCI advice sessions (contact us if you want more information)</li>
-                <li><a class="govuk-link" href="https://landing.google.com/sre/sre-book/chapters/service-level-objectives/#indicators-o8seIAcZ">Implementing service level objectives</a></li>
+                <li>Hide sensitive card details when call centre agents are working from home (a <a class="govuk-link" href="https://www.payments.service.gov.uk/take-payments-by-phone-or-post/">MOTO payments</a> feature)</li>
+                <li>Increased the <a class="govuk-link" href="https://docs.payments.service.gov.uk/custom_metadata/#add-custom-metadata">metadata</a> value limit from 50 to 100 characters</li>
+                <li>Reports on bank payouts from Stripe available via CSV and API</li>
+                <li>Performance improvements to the Transactions page in the <a class="govuk-link" href="https://selfservice.payments.service.gov.uk/login">admin tool</a></li>
+                <li>Various operational improvements and bug fixes</li>
+
               </ul>
 
-             <h2 class="govuk-heading-m">July to September 2020</h2>
+             <h2 class="govuk-heading-m">Things we're working on</h2>
```

Comment:
Do we stick to curly apostrophes as a rule?

Reply:
Oh, @alexbishop1 usually tidies up my use of the wrong apostrophe! But yes, I think we do use `’` not `'`.